src/Whoops/Util/TemplateHelper.php

```diff
@@ -22,6 +22,16 @@ class TemplateHelper
     private $variables = array();
 
     /**
+     * @var HtmlDumper
+     */
+    private $htmlDumper;
```

Comment:
Can you explain why this is better? The TemplateHelper instance is reused during the render step, so it's not necessary to make it a static variable. And in case you wanna render multiple pages, we should have the javascript on each page (which will happen now because we create a new instance in PrettyPageHandler::handle()).